telemetry-sample/src/extension.ts

```diff
@@ -25,14 +21,18 @@ export function activate(context: vscode.ExtensionContext) {
 
 	const c1 = vscode.commands.registerCommand('extension.logEvent', () => {
 		vscode.window.showInformationMessage('Logged telemetry event!');
-		logger.logUsage('testEvent', { 'testProp': 'testValue' });
+		if (logger.isUsageEnabled) {
```

Comment:
I don't even think you technically need these. We should be gating `logger.logUsage` properly via the API to whether or not enablement is present